### DISCUSSION TITLE
Open the expandable with link

### DISCUSSION THREAD

**Discussion by ngnbthuy123131** [GitbookIO/community#604] [Feature Requests]
I have a list of questions created using expandable. I want the expandable will be opened when I send the link of the question to anyone, like [this](https://prnt.sc/Cm0Xf4Aj-fEQ). It would be a better experience for users. Please consider this.

**Comment by oneiric-oskar** [comment]
This should be added for the Tabs block as well. 

In our case we have a problem with search results. If the search term is contained by a tab that is _not active_, you'll end up at the top of the page. This makes it hard to find the section you're looking for, especially if there are a lot of tabs or the page is long. It would improve the user experience if the correct tab was opened, and the browser scrolls down to it.

It seems that GitBook is already aware that the matching text (from the search result) is part of a tab. The tab's title is added to the URL's hash, in the same way as the OP suggests, as far as I can see.

**Comment by aikoii** [comment]
Hey @ngnbthuy123131! You can now open the expandable block with a link! 🎉
